### PR TITLE
Added functionality to check model sizes

### PR DESCRIPTION
**Pull request by TheGogy**
We now have functionality to check model sizes when downloading.
Should look like this:

```
[size] model-name
# e.g:

[2.2 GB] en-large-0.22
[48 MB] en-small-0.15
```

### CODE REVIEW

File: Models/README.md [hunk 1]
```diff
@@ -1,6 +0,0 @@
-# Models
```
[the remark this answers is not in the document]
Comment (box1273002) [reply Models/README.md:1]: didn't need it anyway

**Review by box1273002 — COMMENTED** [review]
Looks good

**Review by TheGogy — COMMENTED** [review]
Looks good!

**Review by box1273002 — APPROVED** [review]
Looks good

**Review by sef007 — APPROVED** [review]

**Pull request merged**